testing/features/support/components/footer/minimal.rb

```diff
@@ -1,15 +1,10 @@
 # frozen_string_literal: true
 
-module DesignSystem
-  class Footer < ::Base
-    set_url "/iframe.html?id=components-footer--default-story&viewMode=story"
-
-    sections :category_titles, ".cads-grid-col-md-3 > h2" do
-      elements :links, :xpath, "./following-sibling::ul[1]/li/a"
-    end
+module Footer
+  class Minimal < ::Base
+    set_url "/iframe.html?id=components-footer--minimal&viewMode=story"
 
     section :initial_form, ".cads-footer" do
```

Comment:
lets remove this wrapping here. And instead just have the components as top level ones. NB: You'll need to edit the below method too.